quickevent/app/quickevent/plugins/Core/src/coreplugin.cpp

```diff
@@ -67,7 +67,7 @@ void CorePlugin::onInstalled()
 	a_file->addSeparatorInto();
 	{
 		auto *a = new qfw::Action(tr("&Settings"));
-		connect(a, &qfw::Action::triggered, this, [=]() {
+		connect(a, &qfw::Action::triggered, this, [=, this]() {
```

Comment:
Are these changes relevant for the PR?

Reply:
Not really. I have fixed this initially because of a warning and I had -Werror on. The project still has warnings. I have kept this change since I've already fixed it

Reply:
Then  I would either mention it in the PR comment or created separate PR for the warning fixes.
Sliding in undocumented code changes can be hard to trace back in case of bugs.

Reply:
I'd say it's documented enough since it's a separate commit message, but you're right, I'll just remove it.